docker/nuxt/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:20-alpine
```

Comment:
Aber du solltest doch eh in der Lage sein node zu verwenden mit dem alpine package. Sonst könntest du ja auch kein `npm install` usw im container ausführen 🤔 Können wir uns anschauen was das Problem ist?

Reply:
wollte git sagen, glaub ich. muss es mir nochmal anschaun